python/lsst/obs/base/gen3/ingest.py

```diff
@@ -353,87 +322,31 @@ def extractDataId(self, file, headers):
 
```

Comment:
obsInfo is not documented.  This seems to be directly derived from the file headers and not modified, but that should still be noted.